clang/docs/ReleaseNotes.rst

```diff
@@ -448,6 +448,11 @@ Modified Compiler Flags
   evaluating to ``true`` and an empty body such as ``while(1);``)
   are considered infinite, even when the ``-ffinite-loop`` flag is set.
 
+- Removed "arm interrupt calling convention" warning that was included in
+  ``-Wextra`` without its own flag.
+
+- Added ``-Warm-interrupt-vfp-clobber``, with its own warning group.
```

Comment:
We should be very clear whether this warning is serving the same function as the one that was removed, or not.

If it is, perhaps it is better to merge the two points so it is clear one is replaced by the other. If not, perhaps you can include a sentence for each like "that would be emitted when..." - "that will be emitted when..." that shows the purpose of each one.

(I know this is in the help, but we can save interested parties a round trip here)

Reply:
Moved these around, and split up the content that was previously in the `Improvements to Clang's diagnostics` section.